Clean orsc directory structures in the repo. Add Capture code and changes to the oRSC BE. New bit files for oRSC BE/FE and CTP6 FE

tsarangi opened the pull request:
Clean orsc directory structures in the repo. Add Capture code and changes to the oRSC BE. New bit files for oRSC BE/FE and CTP6 FEs to the oRSC BE. New bit files for oRSC BE/FE and CTP6 FE

tsarangi committed:
Clean orsc directory structures in the repo. Add Capture code and changes to the oRSC BE. New bit files for oRSC BE/FE and CTP6 FE

tsarangi commented:
@dabelknap 
Can you review this ? I have cleaned up most of the oRSC directory structures. Now all the necessary stuff for oRSC is concentrated into one directory each for BE and FE...

dabelknap commented:
@tsarangi I looked through the code, and at at glance I didn't notice problems. Was there something specific you wanted me to check?

Also, it is difficult to follow the changes you made since you packed everything into a single commit. 93 modified files in a single commit is very large! It would be helpful if in the future you made several, smaller commits of related changes. It is much simpler to follow and understand the changes this way.

tsarangi commented:
@dabelknap 
The changes made earlier are irrelevant and that's why I have removed them all... All the necessary things for oRSC are in one directory each for FE and BE. I have removed your VME stream thing for oRSC, I guess it is no longer needed. Can you confirm ?

dabelknap commented:
VMEStream was for the VMEPC/oRSC interface, which I think has already been handled by yourself and Sridhara. So VMEStream should not longer be needed.
